src/_includes/partials/token_table.njk

```diff
@@ -37,14 +37,15 @@
                     {%- when 'spacing-preview' -%}
                         <div class="spacing-preview" style="width: {{item[1].value}}; height: {{item[1].value}};"></div>
                     {%- when 'token-name' %}
-                        <span lang="en">--gcds-{{ concatToken | replace: ".", "-" | fixTokenName }}-{{ item[0] }}</span>
+                        <span lang="en">--gcds-{{ concatToken | replace: ".", "-" | fixTokenName | camelToKebab }}-{{ item[0] | camelToKebab }}</span>
                     {%- when 'value' %}
                         {{- item[1].value }}
                     {%- when 'contrast' %}
                         {{- item[1].value | contrast }}
                     {%- when 'rgb' %}
                         {{- item[1].value | hexTo: 'rgb' }}
                     {%- when 'px' %}
+                        PX
```

Comment:
```suggestion
```
I think this isn't supposed to be here?
<img width="357" alt="Screen Shot 2024-12-10 at 10 22 26 PM" src="https://github.com/user-attachments/assets/5340ad31-a8ad-48b9-8b60-b58d2d6c6292">

Reply:
Oops good catch! That was from testing and I forgot to remove it - thank you!